### PR TITLE
DAPI-435: add the push structure and product CRON definition into PIM…

### PR DESCRIPTION
**Pull request by rouliane**
… EE installation documentation

**Description**

**Definition Of Done**

| Q                                 | A
| --------------------------------- | ---
| Technical Review and 2 GTM        | Todo
| English Review and 1 GTM          | Todo

`Todo`: Pending / Work in progress
`OK`: Done / Validated
`-`: Not needed

### CODE REVIEW

File: cloud_edition/flexibility_mode/docs/crontasks.rst [hunk 1]
```diff
@@ -73,6 +73,8 @@ If you don't want to use this wrapper you can prepend `SHELL=/bin/bash`, for exa
     0 23 * * * pim:volume:aggregate --env=prod
     #Ansible: pimee:data-quality-insights:schedule-periodic-tasks --env=prod
     15 0 * * * pimee:data-quality-insights:schedule-periodic-tasks --env=prod
+    #Ansible: pimee:franklin-insights:quality-highlights:push-structure-and-products
+    15 0,12 * * * * pimee:franklin-insights:quality-highlights:push-structure-and-products --env=prod
```
Comment (AymericPlanche) [inline cloud_edition/flexibility_mode/docs/crontasks.rst]:
```suggestion
    15 0,12 * * * pimee:franklin-insights:quality-highlights:push-structure-and-products --env=prod
```

File: cloud_edition/flexibility_mode/docs/crontasks.rst [hunk 2]
```diff
@@ -122,18 +124,20 @@ The default crontab at the moment on our Flexibility environments is the followi
...
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:franklin-insights:fetch-products --env=prod                                 | \*/30 \* \* \* \*   | Every 30 minutes         |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| akeneo:reference-entity:refresh-records --all --env=prod                          | 0 23 \* \* \*       | At 11:00 PM              |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:data-quality-insights:schedule-periodic-tasks --env=prod                    | 15 0 \* \* \*       | At 00:15 AM              |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:franklin-insights:quality-highlights:push-structure-and-products --env=prod | 15 0,12 \* \* \* \* | At 12:15 AM and 12:15 PM |
```
Comment (AymericPlanche) [inline cloud_edition/flexibility_mode/docs/crontasks.rst]:
```suggestion
| pimee:franklin-insights:quality-highlights:push-structure-and-products --env=prod | 15 0,12 \* \* \* | At 12:15 AM and 12:15 PM |
```

File: install_pim/manual/crontab_tasks_ee.rst.inc [hunk 3]
```diff
@@ -24,12 +24,13 @@ You have to add these 10 commands to your local installation:
     php /path/to/installation/pim-enterprise-standard/bin/console akeneo:reference-entity:refresh-records --all --env=prod # re-index all reference entities
     php /path/to/installation/pim-enterprise-standard/bin/console pim:volume:aggregate --env=prod # recalculates the number of product values
     php /path/to/installation/pim-enterprise-standard/bin/console pimee:data-quality-insights:schedule-periodic-tasks --env=prod # schedule consolidation and purge of data quality insights related data
+    php /path/to/installation/pim-enterprise-standard/bin/console pimee:franklin-insights:quality-highlights:push-structure-and-products --env=prod #push structure and products data in order to compute the Quality Highlights metrics
```
Comment (AymericPlanche) [inline install_pim/manual/crontab_tasks_ee.rst.inc]: could you differentiate the franklin commands and the regular pim commands (as I did in the 3.2 PR no longer needed https://github.com/akeneo/pim-docs/pull/1285/files#diff-51b17d77a778295676a0b0ffd42f0883R32)

It is more clear that the franklin CRONs are not mandatory for every pim install

**Comment by rouliane** [comment]
@AymericPlanche fixed

File: cloud_edition/flexibility_mode/docs/crontasks.rst [hunk 4]
```diff
@@ -122,18 +124,20 @@ The default crontab at the moment on our Flexibility environments is the followi
...
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:franklin-insights:fetch-products --env=prod                                 | \*/30 \* \* \* \*   | Every 30 minutes         |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| akeneo:reference-entity:refresh-records --all --env=prod                          | 0 23 \* \* \*       | At 11:00 PM              |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:data-quality-insights:schedule-periodic-tasks --env=prod                    | 15 0 \* \* \*       | At 00:15 AM              |
++-----------------------------------------------------------------------------------+---------------------+--------------------------+
+| pimee:franklin-insights:quality-highlights:push-structure-and-products --env=prod | 15 0,12 \* \* \* | At 12:15 AM and 12:15 PM    |
```
Comment (AymericPlanche) [inline cloud_edition/flexibility_mode/docs/crontasks.rst]: the table will not be correctly displayed missing spaces in crontab frequency to correctly align the `|`

**Pull request merged**